projects/demo/src/app/forms/layout-angular/layout-angular.html

```diff
@@ -24,19 +30,43 @@ <h1>Angular - {{layout}} - {{ labelSize ? 'grid' : 'no grid' }}</h1>
     </clr-checkbox-wrapper>
   </clr-checkbox-container>
   <clr-date-container>
-    <label>Date picker</label>
+    <label
+      >Date picker
+      <clr-signpost>
+        <cds-icon shape="info-standard" clrSignpostTrigger>Trigger</cds-icon>
+        <clr-signpost-content *clrIfOpen> Signpost content </clr-signpost-content>
+      </clr-signpost></label
+    >
     <input clrDate />
   </clr-date-container>
   <clr-input-container>
-    <label>Input</label>
+    <label
+      >Input
+      <clr-signpost>
+        <cds-icon shape="info-standard" clrSignpostTrigger>Trigger</cds-icon>
+        <clr-signpost-content *clrIfOpen> Signpost content </clr-signpost-content>
+      </clr-signpost></label
+    >
```

Comment:
```suggestion
    <label>
      Input
      <clr-signpost>
        <cds-icon shape="info-standard" clrSignpostTrigger>Trigger</cds-icon>
        <clr-signpost-content *clrIfOpen>Signpost content</clr-signpost-content>
      </clr-signpost>
    </label>
```

unmangle tags and remove extraneous whitespace